posthog/management/commands/test_migrations_are_safe.py

```diff
@@ -73,7 +73,7 @@ def validate_migration_sql(sql) -> bool:
             )  # Ignore for brand-new tables
         ):
             print(
-                f"\n\n\033[91mFound a CONSTRAINT command. This locks tables which causes downtime. Please avoid adding constraints to existing tables.\nSource: `{operation_sql}`"
+                f"\n\n\033[91mFound a CONSTRAINT command. This locks tables which causes downtime. Please avoid adding constraints to existing tables. For an example of how to do this safely see 0412_pluginconfig_match_action.py . \nSource: `{operation_sql}`"
```

Comment:
I think you want this on line 85 as well - where we have the create index concurrently check.

The NULL constraint I think is just boilerplate, I'd say we can go as far as removing it from this check, and in that case the error would be on the create index command.

Reply:
<img width="1372" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/be9286b6-fb8f-46b9-a736-378cfb71231c">

Reply:
Added it in the other place too